models/user/src/user/types.ts

```diff
@@ -0,0 +1,31 @@
+import type { TBuilder } from '@commercetools-test-data/core';
+import { TCoreUser } from '@commercetools-test-data/graphql-types/src/generated/core';
+
+type BusinessRole =
+  | 'ExecutiveManagement'
+  | 'CustomerService'
+  | 'Marketing'
+  | 'SalesAndECommerceManager'
+  | 'ProductProjectManagerOrOwner'
+  | 'Architect'
+  | 'Engineer'
+  | 'Other';
```

Comment:
nit: just wondering, couldn't we get this type from the generated types?

Reply:
Yes, we can. Thanks. [f458f92](https://github.com/commercetools/test-data/pull/466/commits/f458f92496e67a42bc6e90814435d17f7f575693)